### ISSUE TITLE
There is no Obsidian Git plugin

### ISSUE THREAD

**Issue by johnblommers** [snazzybytes/obsidian-scripts#1]
I've searched both the core and the community plugins for Obsidian and the is no plugin called Obsidian Git to be seen.

Can you be more specific where this plugin might be found?

**Comment by snazzybytes** [comment]
You have to go to COMMUNITY instead of core plugins. It's literally called "Obsidian Git", I am guessing you looked in CORE right? Settings>Community instead 🤙

https://github.com/denolehov/obsidian-git

**Comment by snazzybytes** [comment]
And I only use it on Laptop/Desktop. On android, instead of Obsidian Git I use those shell scripts and Termux instead. Desktop vs Mobile fyi

**Comment by snazzybytes** [comment]
Good point though @johnblommers I will update readme so it's clear it's a Community not a Core plugin. Thanks

**Comment by WybeBosch** [comment]
I think on android you can also turn on community plugins these days so you can use the Obsidian Git Community plugin on android aswell.  And you only need termux to setup the initial git repo with ssh keys / decrypt stuff but no longer for the desktop push pull widgets.

(On android the Obsidian Git plugin doesnt support ssh repo's tho, need to be https which is annoying)
